### PR TITLE
Better GitHub issue templates

### PR DESCRIPTION
**Pull request by fhemberger**
The [babel](https://github.com/babel/babel/issues/new/choose) project had a pretty neat idea for GitHub issue templates, so I'd like to adapt them for the Node.js website.

As we receive issues for API docs (which are not hosted here) and other, it might be a good idea to guide the user to the right place directly.

### CODE REVIEW

**Review by tniessen — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,22 +1,6 @@
...
-For general questions about Node.js:
-https://github.com/nodejs/help/issues/
-
-Please fill in the template below by replacing the HTML comments with an
-appropriate answer. If unsure about something, just do as best as you're able.
-If you are reporting a visual glitch, it will be much easier for us to fix it
-when you attach a screenshot as well.
-
-Thank you!
+Check the other issue templates if you are trying to submit a bug report, feature request, or question
```
Comment (vsemozhetbyt) [inline .github/ISSUE_TEMPLATE.md]: A period in the end?

File: .github/ISSUE_TEMPLATE/api_docs.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+---
+name: 📗 Node.js API Docs
+about: Please open an issue in the main [Node.js repo](https://github.com/nodejs/node/issues/), prefixed with "doc".
+
+---
+<!--
+For issues regarding the Node.js API docs, please head over to:
+https://github.com/nodejs/node/issues/ (prefix your issue name with "doc")
```
Comment (vsemozhetbyt) [inline .github/ISSUE_TEMPLATE/api_docs.md]: Ditto?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+about: I have a suggestion (and may want to implement it 🙂)!
+
+---
+<!--
+You have an idea how to improve the site? That's awesome!
+
+Before submitting, please have a look at the existing issues if there's already
+something related to your suggestion.
+
+We are also [working on a relaunch](https://github.com/nodejs/website-redesign/issues/)
```
Comment (vsemozhetbyt) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Is it worth to replace the markdown markup in the comment with a more readable plain text?

File: .github/ISSUE_TEMPLATE/help.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+name: ⁉️ General Questions
+about: Need help with Node.js? File an Issue [here](https://github.com/nodejs/help/issues/).
+
+---
+<!--
+We have a special help section for general quesitons about Node.js:
```
Comment (vsemozhetbyt) [inline .github/ISSUE_TEMPLATE/help.md]: quesitons -> questions.

**Comment by fhemberger** [comment]
@vsemozhetbyt Thanks, fixed!

**Review by vsemozhetbyt — APPROVED** [review]

**Review by stevemao — APPROVED** [review]

**Review by ckotzbauer — APPROVED** [review]

**Review by Tsuna-mi — APPROVED** [review]

**Pull request merged**